dCommon/dEnums/ePropertySortType.h

```diff
@@ -0,0 +1,11 @@
+#ifndef EPROPERTYSORTTYPE_H
+#define EPROPERTYSORTTYPE_H
```

Comment:
I think you told me this earlier, but why aren't we just using `#pragma once` here?

Reply:
doesn't work with pre-compiled headers on gcc